doc/api/permissions.md

```diff
@@ -440,7 +446,157 @@ not adopt the origin of the `blob:` URL.
 Additionally, import maps only work on `import` so it may be desirable to add a
 `"import"` condition to all dependency mappings.
 
+## Process-based permissions
+
+### Permission Model
+
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+<!-- name=permission-model -->
+
+The Node.js Permission Model is a mechanism to restrict access to specific
+resources during the program execution.
+The API exists behind a flag [`--experimental-permission`][] which when enabled,
+will restrict access to all available permissions.
+
+Currently, the available permissions are:
+
+* File System - manageable through [`--allow-fs`][] flag
+* Child Process - manageable through [`--allow-child-process`][] flag
+* Worker Threads - manageable through [`--allow-worker`][] flag
+
+Therefore, when starting a Node.js process with `--experimental-permission`,
+the capabilities to access the filesystem, spawn process and,
+use `node:worker_threads` will be restricted.
+
+```console
+$ node --experimental-permission index.js
+node:internal/modules/cjs/loader:171
+  const result = internalModuleStat(filename);
+                 ^
+
+Error: Access to this API has been restricted
+    at stat (node:internal/modules/cjs/loader:171:18)
+    at Module._findPath (node:internal/modules/cjs/loader:627:16)
+    at resolveMainPath (node:internal/modules/run_main:19:25)
+    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:24)
+    at node:internal/main/run_main_module:23:47 {
+  code: 'ERR_ACCESS_DENIED',
+  permission: 'FileSystemIn'
+}
+```
+
+Allowing access to spawning a process and creating worker threads can be done
+using the `--allow-child-process` and `--allow-worker` respectively.
+
+#### Runtime API
+
+When enabling the Permission Model through the [`--experimental-permission`][]
+flag a new property `permission` is added to the `process` module.
+This property contains two functions:
+
+* `permission.deny(scope [,parameters])`
+  * `scope` {string}
+  * `parameters` {Array} array of strings
+  * Returns: {boolean} Operation result.
```

Comment:
I'm not sure that we need to reiterate this here. Can we just link to the appropriate place in the `process` docs?